int32 id
geometry_msgs/Pose  pose
bool[] sensor_status